src/api/index.ts

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export const axiosInstance = axios.create({
+  baseURL: 'https://react-gift-mock-api-sigma.vercel.app/',
+  timeout: 10000,
```

Comment:
시간이 기본적으로 ms 단위이기 때문에 이런 식으로 식별하기 좋게 적어주는 경우도 많으니 참고 하시면 좋을거 같네요

```suggestion
  timeout: 10 * 1_000,
```

Reply:
numeric seperator와 시간을 연산자로 분리하여 표현하는 방법을 사용하니 가독성이 확 올라간 것 같습니다! 앞으로 큰 숫자 및 시간 관련 코드를 짤 때 활용해보겠습니다!!